tests/tt_eager/python_api_testing/unit_testing/backward_ops/test_backward_add.py

```diff
@@ -34,3 +34,45 @@ def test_bw_add(input_shapes, device):
 
     status = compare_pcc(tt_output_tensor_on_device, golden_tensor)
     assert status
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+@pytest.mark.parametrize("are_required_outputs", [[True, True], [True, False], [False, True], [False, False]])
```

Comment:
There's no need to test the case where both requires_grad are False. Therefore, please remove this case.
```suggestion
@pytest.mark.parametrize("are_required_outputs", [[True, True], [True, False], [False, True])
```

Reply:
ok, will remove that case